src/components/tipRecords/TipRecord.vue

```diff
@@ -39,6 +39,7 @@
           <div
             class="tip__comments"
             :class="[{ 'tip__comments--hascomments': tip.commentCount }]"
+            @click.stop="$route.params.tipId === tip.id ? null : goToTip()"
```

Comment:
- I think to refer by route name is clearer
- can we write it without binding the same handler twice in the case of tip list?